middleware.ts

```diff
@@ -0,0 +1,15 @@
+
+import { NextResponse } from "next/server";
+import { NextRequest } from "next/server";
+
+export async function middleware(req: NextRequest) {
+  // TODO: see why this isn't working in async redirects in next.js config
+  if (req.nextUrl.pathname === "/") {
+    return NextResponse.redirect(
+      new URL(
+        '/teams',
+        req.url
+      )
+    );
+  }
+}
```

Comment:
nit: missing newline at end of file?

Reply:
Prettier in this project must not be configured that way. Will fix.